keps/NNNN-kep-template/README.md

```diff
@@ -332,6 +342,151 @@ enhancement:
   CRI or CNI may require updating that component before the kubelet.
 -->
 
+## Production Readiness Review Questionnaire
+
+<!--
+
+Production readiness reviews are intended to ensure that features merging into
+Kubernetes are observable, scalable and supportable, can be safely operated in
+production environments, and can be disabled or rolled back in the event they
+cause increased failures in production. See more in the PRR KEP at
+https://git.k8s.io/enhancements/keps/sig-architecture/20190731-production-readiness-review-process.md
+
+The KEP must have a approver from the PRR review team.
```

Comment:
Regarding the tooling, I have no idea how the existing tooling works, so I may be completely wrong, but I think it hopefully should be a small thing to do. How I envision this to work is:
1. add another PRR_OWNERS file (or extend the existing one - thanks @johnbelamaric for this thought) [we can probably start with just top-level one
2. introduce prr-approved label (naming to be done) based on /prr-approve comment from PRR_OWNERS (hopefully that's only a matter of configuring existing tooling?)
3. change tide for this repo to require both approve and prr-approve labels?

As @johnbelamaric pointed out we will probably need to also auto-assign prr-approvers. Though I think that as a first step we can probably ask sig-leads (who are the only approvers for now) to add someone manually.
The remaining thing is that not every PR should require PRR-review. But OTOH, those trivial ones will probably be easy to review and will not add significant overhead.

@spiffxp @cblecker - does that above makes any sense from the tooling perspective? How much work would it be to make that happen? [If you're not the right people to answer, can you route it to the right ones?]

Reply:
I'm not sure I'm a fan of a coding something specifically into prow for this purpose. I agree we're seeing cases for more flexible review/approval workflows than what we support right now, but it's unclear what the right path forward is, and nobody has improving this as a priority. Need some time to think about this. @cjwagner @fejta do you have any ideas?

A mechanism you could try to use today is to force KEPs going to release to use a directory based approach (I'm not clear if that's the standard now or if we're still putting everything in one file), put the PRR in subdir of the KEP, and use a no_parent_owners directive + approves to restrict approval of the PRR to your team/alias.

Reply:
That OWNERS file will be created by the KEP author - can we somehow enforce that it will be added and will have the no-parent stuff?

Reply:
We can definitely add something as a presubmit to enforce this!

Reply:
I'd agree with @spiffxp that no_parent_owners is favorable over creating a new workflow here.

Reply:
I'd agree with @spiffxp that no_parent_owners is favorable over creating a new workflow here.